### DISCUSSION TITLE
Window icons next to their workspace icons to show apps open present in those workspace.

### DISCUSSION THREAD

**Discussion by SherLock707** [JaKooLit/Hyprland-Dots#105] [General]
Something like this:
![image](https://github.com/JaKooLit/Hyprland-Dots/assets/26952545/8f06e73e-4db8-4647-a924-b0de2452edc1)

I couldn't find a way to do this.

App icons can be clickable or not.

wlr/taskbar is bit janky and feels disconnected from workspace...
![image](https://github.com/JaKooLit/Hyprland-Dots/assets/26952545/40351693-42e7-4281-b368-e6b0ca86c4f3)

**Comment by JaKooLit** [comment]
I have that already... I mean waybar module is there....

you just need to choose it

**Reply by SherLock707** [reply]
`hyprland/workspaces#4`

![image](https://github.com/JaKooLit/Hyprland-Dots/assets/26952545/6dab29c8-ff0e-45fc-8ae0-c9ae52877047)

This one?

This one is static icons, I was wondering if there is a way to add icons based on whats opened in that workspace dynamically.

[1: (Firefox icon) (kitty) 2: (Lutris) (Thunar) 3: 4:(VSCode)]
